api/tests/unit/features/workflows/core/test_unit_workflows_models.py

```diff
@@ -897,3 +977,31 @@ def _create_segment(percentage_value: int) -> Segment:
     )
     assert len(after_cr_1_flags) == 1
     assert after_cr_1_flags[0].feature_segment.segment == twenty_percent_segment
+
+
+def test_approval_via_project(project_change_request: ChangeRequest) -> None:
+    # Given - The project change request fixture
+    assert project_change_request.environment is None
+    assert project_change_request.project.minimum_change_request_approvals is None
+
+    # When
+    is_approved = project_change_request.is_approved()
+
+    # Then
+    assert is_approved is True
```

Comment:
Is this a legit use case? For environments at least, `minimum_change_request_approvals = None` means that change requests aren't enabled at all.

Reply:
According to a pre-existing test (`test_change_request_is_approved_returns_true_when_minimum_change_request_approvals_is_none`) I believe your understanding is incorrect. I just mirrored the behavior from the environments definition for the project piece.